hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/keys/Ed25519Utils.java

```diff
@@ -73,12 +88,12 @@ public static EdDSAPrivateKey readKeyFrom(final File pem, final String passphras
         }
     }
 
-    public static Path relocatedIfNotPresentInWorkingDir(final Path p) {
-        return relocatedIfNotPresentInWorkingDir(p.toFile()).toPath();
+    public static Path relocatedIfNotPresentInWorkingDir(final Path path) {
+        return relocatedIfNotPresentInWorkingDir(path.toFile()).toPath();
```

Comment:
Changes like this also seem excessive

Reply:
Please see https://google.github.io/styleguide/javaguide.html#s5.2.6-parameter-names